app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventDetailsFragment.java

```diff
@@ -69,61 +72,23 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
      */
     private void showDetails(LinearLayout rootView) {
         url = event.getLink();
-
         createEventHeader(rootView);
         createEventFooter(rootView);
     }
 
-    private void addToRoot(LinearLayout rootView, int headerId, CharSequence contentString) {
-        View view = inflater.inflate(R.layout.list_header_big, rootView, false);
-        TextView text = view.findViewById(R.id.list_header);
-        text.setText(headerId);
-        rootView.addView(view);
-        view = inflater.inflate(R.layout.kino_content, rootView, false);
-        text = view.findViewById(R.id.line_name);
-        text.setText(contentString);
-        rootView.addView(view);
-    }
-
-    private void addToRootWithPadding(LinearLayout rootView, int headerId, CharSequence contentString) {
-        View view = inflater.inflate(R.layout.list_header_big, rootView, false);
-        TextView text = view.findViewById(R.id.list_header);
-        text.setText(headerId);
-        rootView.addView(view);
-        view = inflater.inflate(R.layout.kino_content, rootView, false);
-        text = view.findViewById(R.id.line_name);
-        text.setText(contentString);
-        // padding is done programmatically here because we need more padding at the end
-        int padding = (int) context.getResources()
-                                   .getDimension(R.dimen.padding_kino);
-        int paddingRight = (int) context.getResources()
-                                        .getDimension(R.dimen.padding_kino_right);
-        int paddingEnd = (int) context.getResources()
-                                      .getDimension(R.dimen.padding_kino_end);
-        text.setPadding(padding, padding, paddingRight, paddingEnd);
-        rootView.addView(view);
-    }
-
-    private void createEventFooter(LinearLayout root) {
-        addToRoot(root, R.string.date, new SimpleDateFormat("yyyy-MM-dd hh:mm", Locale.GERMANY).
-                format(event.getDate()));
-        addToRoot(root, R.string.location, event.getLocality());
-        addToRootWithPadding(root, R.string.description, event.getDescription());
-    }
 
     private void createEventHeader(LinearLayout rootView) {
         LinearLayout headerView = (LinearLayout) inflater.inflate(R.layout.event_header, rootView, false);
 
         // initialize all buttons
-        Button link = headerView.findViewById(R.id.button_link);
+        //Button link = headerView.findViewById(R.id.button_link);
```

Comment:
Remove this if we do not use it anymore
Code which is disabled might confuse developsers in the future